### PR TITLE
Complete Design for Login Page

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: src/client/App.css [hunk 1]
```diff
@@ -122,36 +143,37 @@ h2 {
...
 
-#registrationForm .passwordLabel {
+.passwordLabel {
   grid-area: passwordLabel;
 }
-#registrationForm #password {
+#password {
   grid-area: password;
   font-weight: 100;
 }
```
Comment (nathanSamosa) [inline src/client/App.css:159]: I like how you used more global styles for forms so they can be reused

File: src/client/App.css [hunk 2]
```diff
@@ -122,36 +143,37 @@ h2 {
...
+  grid-area: submit;
   margin-top: 20px;
+  margin-bottom: 10px;
+  height: 25px;
   background: #ffffff;
   border: none;
-  margin-bottom: 10px;
-  height: 30px;
   box-shadow: -2px -2px 5px #8ec5d18f, 
               2px 2px 5px #e9ab7d8f
```
Comment (nathanSamosa) [inline src/client/App.css:178]: I didnt know you could format shadows like this, does it create a 50/50 gradient?

File: src/client/App.css [hunk 3]
```diff
@@ -41,12 +41,16 @@ ul {
   list-style: none;
 }
 
-.registrationPage h1 {
-  margin-bottom: 10vh;
+.loginPage h1 {
```
Comment (TheunisK) [inline src/client/App.css:44]: Why did you change this to loginPage? Will is not interfere with the registration styling?

File: src/client/App.jsx [hunk 4]
```diff
@@ -1,11 +1,11 @@
 import './App.css';
 
-import Registration from './components/Registration';
+import Login from './components/Login';
 
 export default function App() {
   return (
     <div className="App">
-      <Registration />
+      <Login />
```
Comment (nathanSamosa) [inline src/client/App.jsx:8]: Just to be clear, we are only importing/removing imports like this for testing before routing is implemented?

Comment (TheunisK) [inline src/client/App.jsx:8]: What happened to the Registration component?

File: src/client/App.css (hunk 3 above)
Comment (Can-Tina) [reply src/client/App.css:44]: Registration page does not have an h1 anymore, this was left over from when it is and I simply reused what was already there for the login page

File: src/client/App.jsx (hunk 4 above)
Comment (Can-Tina) [reply src/client/App.jsx:8]: I only put the registration and app components in there for testing. Since I don't know yet how we're going to do the pages/routing, I removed the registration

File: src/client/App.css (hunk 1 above)
Comment (Can-Tina) [reply src/client/App.css:159]: Thank you :)

File: src/client/App.css (hunk 2 above)
Comment (Can-Tina) [reply src/client/App.css:178]: Sort of, yes. The buttons have blue on the left and the top and orange on the right and the bottom which seems to be the default for 2 colours

File: src/client/App.jsx (hunk 4 above)
Comment (Can-Tina) [reply src/client/App.jsx:8]: Yes, that was my plan. I just need to be able to see what I'm doing is all

**Comment by glowkeeper** [comment]
Approving
